### PR TITLE
Correct racial mod stats, table formatting and annotations, dual classing added to human class table

### PR DESCRIPTION
**Pull request by RandomContributions**

### CODE REVIEW

File: src/rules.c [hunk 1]
```diff
@@ -38,102 +39,333 @@ static int convert_to_class_sel(const uint8_t class) {
 }
 
 static const uint16_t human_classes[] = {
-    A_C, A_D, A_F, A_G, A_P, A_S, A_R, A_T, A_END
+//  CLR     DRU     FGR     GLD     PRE     PSI     RNG     THF
+    A_C,    A_D,    A_F,    A_G,    A_P,    A_S,    A_R,    A_T,
+
+
+//  DUAL  CLR / FGR     DUAL  CLR / GLD     DUAL  CLR / PRE     DUAL  CLR / PSI
+    A_U | A_C | A_F,    A_U | A_C / A_G,    A_U | A_C | A_P,    A_U | A_C | A_S,
```
Comment (dsoageofheroes) [inline src/rules.c]: Shouldn't that be:
A_U | A_C | A_F,    A_U | A_C | A_G,    A_U | A_C | A_P,    A_U | A_C | A_S,
The written one has a '/'

File: src/rules.c [hunk 2]
```diff
@@ -146,266 +378,265 @@ static void dnd2e_apply_race_mods(ds_character_t *pc) {
...
-    {6, 8, 75, 14, 0},
-    {7, 9, 85, 18, 0}, // 18
-    {8, 9, 95, 9999, 1},
-    {9, 9, 96, 9999, 2},
-    {10, 9, 97, 9999, 3},
-    {11, 9, 98, 9999, 4},
-    {12, 9, 99, 9999, 5},
-    {15, 9, 100, 9999, 6},
-    {20, 9, 100, 9999, 7},
+    {    0,          0,        0,            0,            0 }, // 00 (should be character death?)
```
Comment (dsoageofheroes) [inline src/rules.c:421]: Yes, it should be character death, but that will be handled elsewhere. We do need an entry in the table for easy lookup. Think of it as a starting buffer.

**Review by dsoageofheroes — CHANGES_REQUESTED** [review]
I think there was a type-o

File: src/rules.c (hunk 1 above)
Comment (RandomContributions) [reply src/rules.c]: whoops, yes. thanks. fixing.

File: src/rules.c (hunk 2 above)
Comment (RandomContributions) [reply src/rules.c:421]: Removed that portion of the comment.

**Pull request merged**